src/nbl/video/CVulkanPhysicalDevice.h

```diff
@@ -888,6 +1108,37 @@ class CVulkanPhysicalDevice final : public IPhysicalDevice
             addFeatureToChain(&bufferDeviceAddressFeatures);
         }
 
+        // Always enable these when present, reported as limit later
```

Comment:
You didn't have to actually write the code, I will have to change most of this again later 
https://discord.com/channels/593902898015109131/723305695046533151/995964241343422484
Anyways I added this to your to-do:
```
make sure features that are moved to limits are still commented in the SPhysicalDeviceFeatures but with an extra comment: // Enabled by Default, Moved to Limits 

```
whatever you removed from features bring it back and add the comments for each one.